android/app/src/main/java/com/papel/Constants.java

```diff
@@ -1,7 +1,8 @@
 package com.papel;
 
 public class Constants {
-    public static final String LOCALHOST = "http://ec2-18-197-152-183.eu-central-1.compute.amazonaws.com:3000/";
+    //public static final String LOCALHOST = "http://ec2-18-197-152-183.eu-central-1.compute.amazonaws.com:3000/";
+    public static final String LOCALHOST = "http://192.168.2.188:3000/";
```

Comment:
Can you remove your local ip and uncomment the other host?

Reply:
Thank you for the feedback.I fix it.